weldx/transformations/local_cs.py

```diff
@@ -548,7 +549,7 @@ def is_time_dependent(self) -> bool:
             `True` if the coordinate system is time dependent, `False` otherwise.
 
         """
-        return self.time is not None or self._coord_ts is not None
+        return self._coord_ts is not None or ("time" in self._dataset.dims)
```

Comment:
I think this is the most important change, a LCS can be time independent but still return a `.time` value